### PR TITLE
updating menu click syntax highlighting

### PR DESCRIPTION
**Pull request by kalexand-rh**
@openshift/team-documentation, will you PTAL?

### CODE REVIEW

**Comment by mburke5678** [comment]
@kalexand-rh Is the `$$` an escape character to retain the `*` in the text? If so, LGTM

**Comment by bergerhoffer** [comment]
@kalexand-rh Thank you for this! I've seen a lot of places where the asterisks only surround the outside of the whole path and that's bothered me :). Also, at least the guidance already has this, but I've seen places that use an arrow character instead of "->", so we might want to point this out as well.

**Comment by kalexand-rh** [comment]
@mburke5678, you're right. I think that I need to escape more characters now. (If it doesn't look right when I merge, I'll drop in another fix.)

@bergerhoffer, me, too! I've seen a mix of use, but I think that enclosing the UI elements in asterisks individually will help us update the docs when the UI changes. I added more guidance for the arrow.

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -725,16 +725,17 @@ a|Use the following command to roll back a Deployment, specifying the Deployment
 `oc rollback <deployment>`
 
 |Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.).
+Use two characters to form the arrow in a series of menu items, $$->.
```
Comment (bergerhoffer) [inline contributing_to_docs/doc_guidelines.adoc]: The "$$" is rendering here. You need to add "$$" after the arrow characters.

File: contributing_to_docs/doc_guidelines.adoc [hunk 2]
```diff
@@ -725,16 +725,17 @@ a|Use the following command to roll back a Deployment, specifying the Deployment
...
 
 a|Choose $$*Cluster Console*$$ from the list.
 
-Navigate to the $$*Operators -> Catalog Sources*$$ page.
+Navigate to the $$*Operators* $$-> $$*Catalog Sources*$$ page.
```
Comment (bergerhoffer) [inline contributing_to_docs/doc_guidelines.adoc]: Actually, adding the $$'s in the middle there is causing the arrow to render. You should only need the $$'s surrounding the entire thing:

```
"$$*Operators* -> *Catalog Sources*$$"
```

**Review by bergerhoffer — COMMENTED** [review]
The recent updates caused just a few issues.

**Pull request merged**